yolo_model:

  config_file:
    name: yolov4-custom.cfg
  weight_file:
    name: yolov4-custom_final.weights
  threshold:
    value: 0.3
  detection_classes:
    names:
      - Red
      - Yellow
      - Green
      - Red_arrow
      - Yellow_arrow
      - Green_arrow